images-list

```diff
@@ -972,6 +972,8 @@ rancher/coreos-flannel rancher/mirrored-coreos-flannel v0.12.0
 rancher/metrics-server rancher/mirrored-metrics-server v0.3.4
 rancher/metrics-server rancher/mirrored-metrics-server v0.3.6
 rancher/nginx-ingress-controller-defaultbackend rancher/mirrored-nginx-ingress-controller-defaultbackend 1.5-rancher1
+registry.k8s.io/sig-storage/snapshot-controller rancher/mirrored-sig-storage-snapshot-controller v6.1.0
+registry.k8s.io/sig-storage/snapshot-validation-webhook rancher/mirrored-sig-storage-snapshot-validation-webhook v6.1.0
```

Comment:
Links for repos we are waiting on
https://hub.docker.com/r/rancher/mirrored-sig-storage-snapshot-controller
https://hub.docker.com/r/rancher/mirrored-sig-storage-snapshot-validation-webhook

Reply:
Repos are now present.